src/version.cmake

```diff
@@ -0,0 +1,6 @@
+EXECUTE_PROCESS(
```

Comment:
I tried to use EXECUTE_PROCESS myself but the output from git was truncated (instead of "0.9-19324-g874cb5d-dirty" i got only "74cb5d-dirty" part). So i tried to use EXECUTE_PROGRAM and it worked. I'm using CMake 3.1.3 btw.